images/update-dbg/dbg-go/pkg/autogenerate/types.go

```diff
@@ -0,0 +1,51 @@
+package autogenerate
+
+import "github.com/falcosecurity/test-infra/images/update-dbg/dbg-go/pkg/root"
+
+type Options struct {
+	root.Options
+	Architecture  string
+	DriverVersion []string
+	OutputRoot    string
+}
+
+type KernelEntry struct {
+	KernelVersion    string   `json:"kernelversion"`
+	KernelRelease    string   `json:"kernelrelease"`
+	Target           string   `json:"target"`
+	Headers          []string `json:"headers"`
+	KernelConfigData []byte   `json:"kernelconfigdata"`
+}
+
+// Please add new supporteed build-new-drivers structures here,
+// so that the utility starts building configs for them.
+// Fields must have the same name used in kernel-crawler json keys.
+type KernelList struct {
```

Comment:
Another nice side-effect: we will only build configs for supported distros.
Note: this list should always match the `build-drivers` jobs configs, ie: the distro we want to support prebuilt artifacts for.